packages/beacon-node/package.json

```diff
@@ -131,6 +131,7 @@
     "@multiformats/multiaddr": "^11.0.0",
     "@types/datastore-level": "^3.0.0",
     "buffer-xor": "^2.0.2",
+    "c-kzg": "^1.0.9",
```

Comment:
@g11tech  Do we want to install c-kzg always now? Current strategy was to keep as optional dependency right?

Reply:
yes, we can install it always :+1:

Reply:
Are you sure? deneb is not necessary now and it can crash Lodestar installations